segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/metadata/ChunkMetadataStoreTests.java

```diff
@@ -941,6 +944,50 @@ public void testEvictionFromBuffer() throws Exception {
         }
     }
 
+    @Test
+    public void testReadWriteFailureAfterEvictionFromBuffer() throws Exception {
+        if (metadataStore instanceof InMemoryMetadataStore) {
```

Comment:
Nit: if the whole test is one big `if` statement, consider reversing the condition and exiting earlier. Otherwise the whole method will be indented for no reason.

Also, is there a way to say "Assert.inconclusive` or whatever to indicate that the test did not run?

Reply:
fixed.